vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/pipeline_embedding_info.cc

```diff
@@ -0,0 +1,83 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/ml/pipeline/pipeline_embedding_info.h"
+
+#include <utility>
+#include <vector>
+
+#include "base/time/time.h"
+
+namespace ads {
+namespace ml {
+namespace pipeline {
+
+EmbeddingPipelineInfo::EmbeddingPipelineInfo() = default;
+
+EmbeddingPipelineInfo::EmbeddingPipelineInfo(
+    EmbeddingPipelineInfo&& info) noexcept = default;
+
+EmbeddingPipelineInfo& EmbeddingPipelineInfo::operator=(
+    EmbeddingPipelineInfo&& info) noexcept = default;
+
+EmbeddingPipelineInfo::~EmbeddingPipelineInfo() = default;
+
+bool EmbeddingPipelineInfo::FromValue(base::Value value) {
```

Comment:
Can we please change this param to `base::Value::Dict` to match modernized usage

Reply:
@tmancey I updated this method to take `base::Value::Dict& value`, rather than `const base::Value::Dict& value`. Adding `const` caused issues with calls to deleted constructors on `item.second`

Reply:
Resolved